pkg/virt-handler/vm_test.go

```diff
@@ -2504,58 +2504,6 @@ var _ = Describe("VirtualMachineInstance", func() {
 				err := controller.checkNetworkInterfacesForMigration(vmi)
 				Expect(err).To(HaveOccurred())
 			})
-			It("should block migration for VMI with SRIOV interface when feature-gate SRIOVLiveMigration is off", func() {
```

Comment:
do we want to add a negative test, to make sure that even if there is SRIOVLiveMigration=off then it works?

Reply:
With these PR changes SRIOVLiveMigration does not exist anymore, SR-IOV Live Migration will be controlled by LiveMigration feature gate.

Reply:
I'll rephrase. Don't we want a test where we migrate a VMI with sriovInterfaceName (naturally without the SRIOVLiveMigration flag cause it's irrelevant anymore), ? 
I don't see we have this kind of test on this go file.

Reply:
Yes, there is a test from SR-IOV migration:
https://github.com/kubevirt/kubevirt/blob/main/tests/network/sriov.go#L441

Reply:
ah then I'm good